### PR TITLE
Added Dnotitia

### PR DESCRIPTION
**Pull request by hello-dnotitia**
Added Dnotitia

### CODE REVIEW

**Comment by aolofsson** [comment]
Do you/they design their own silicon. Not clear.

File: README.md [hunk 1]
```diff
@@ -116,6 +116,7 @@
 |[DeGirum](https://degirum.ai) | AI | 2017 | US |AI hardware and software to accelerate AI edge adoption |
 |[Denpaflux](https://denpaflux.com) | EDA | 2022 | DE |AI based PCB design |
 |[Dfiant](https://dfiant.works) | EDA | 2022 | US |Chip design tools |
+|[Dnotitia](https://dnotitia.com) | AI | 2023 | KR |Vector DPUs |
```
Comment (afaerber) [inline README.md:119]: Your entry's description is getting cut off at the first comma ...

File: startups.csv [hunk 2]
```diff
@@ -57,6 +57,7 @@ d-Matrix,d-matrix.ai,AI,US,2019,Computing platform for data center AI inference
 DeGirum,degirum.ai,AI,US,2017,AI hardware and software to accelerate AI edge adoption
 Denpaflux,denpaflux.com,EDA,DE,2022,AI based PCB design
 Dfiant,dfiant.works,EDA,US,2022,Chip design tools
+Dnotitia,dnotitia.com,AI,KR,2023,Vector DPUs, LLM systems, and HW/SW solutions for generative AI and AGI.
```
Comment (afaerber) [inline startups.csv:60]: ... you need quotes or some other escape mechanism for commas in the Comma Separated Values format.

Comment (afaerber) [inline startups.csv:60]: The website actually spells it as "D.notitia", whereas here you spell it consistently without the dot.

**Review by afaerber — CHANGES_REQUESTED** [review]
Note that while the pull request has been renamed, the Git commit still misleadingly says "updated" although it's adding a new entry.

All that said, I second @aolofsson's concerns: "Database" on the website sounds like software?

**Comment by aolofsson** [comment]
closing due to lack of response

**Pull request closed without merging**